tests/statemodels_test.py

```diff
@@ -17,4 +17,4 @@ def test_add_history_should_set_proper_timestamp(self, fake_event):
 
 @pytest.fixture
 def fake_event():
-    yield Event(id=42, router="example-gw.example.org", event_type=EventType.REACHABILITY, state=EventState.OPEN)
+    yield Event(id=42, router="example-gw.example.org", event_type=ReachabilityEvent, state=EventState.OPEN)
```

Comment:
```suggestion
    yield Event(id=42, router="example-gw.example.org", state=EventState.OPEN)
```

Reply:
You're commenting on something that was part of #19 here - it is no longer part of this PR...